docs/docs/components/guidelines/README.md

```diff
@@ -373,13 +373,13 @@ Examples:
 - The async option to select an Airtable Base is self-explanatory so includes no
   description:
 
-  ![image-20210326151557417](images/image-20210326151557417.png)
+  ![image-20210326151557417](../../../../images/image-20210326151557417.png)
 
 - The “Search Term” prop for Twitter includes a description that helps the user
   understand what values they can enter, with specific values highlighted using
   backticks and links to external content.
 
-  ![image-20210326151706682](images/image-20210326151706682.png)
+  ![image-20210326151706682](../../../../images/image-20210326151706682.png)
```

Comment:
You'll also need to move these images into the local `images/` subdirectory (see main comment on why the Vercel build failed)